contracts/src/bridge/Outbox.sol

```diff
@@ -29,8 +29,20 @@ contract Outbox is DelegateCallAware, IOutbox {
     L2ToL1Context internal context;
     uint128 public constant OUTBOX_VERSION = 2;
 
+    //workaround for constant struct
+    function getDefaultContext() internal pure returns (L2ToL1Context memory) {
```

Comment:
lets add some [natspec](https://docs.soliditylang.org/en/v0.8.13/natspec-format.html#documentation-example) to explain the function
```suggestion
    /// @dev returns default values to be used in storage instead of zero, to save on storage refunds
    /// @dev solidity doesn't support constant structs, so instead we return it in a pure function instead
    /// @dev it is assumed that arb-os never assigns these values to a valid leaf
    function getDefaultContext() internal pure returns (L2ToL1Context memory) {
```